packages/storage-plus/src/indexes/multi.rs

```diff
@@ -22,19 +22,17 @@ use std::marker::PhantomData;
 /// The stored pk_len is used to recover the pk from the index namespace, and perform
 /// the secondary load of the associated value from the main map.
 ///
-/// The MultiIndex definition must include a field for the pk. That is, the MultiIndex K value
-/// is always a n-tuple (n >= 2) and its last element must be the pk.
-/// The index function must therefore put the pk as last element, when generating the index.
-/// The optional PK type defines the type of Primary Key deserialization.
-pub struct MultiIndex<'a, K, T, PK = ()> {
-    index: fn(&T, Vec<u8>) -> K,
+/// The (optional) PK type defines the type of Primary Key deserialization.
+pub struct MultiIndex<'a, IK, T, PK = ()> {
+    index: fn(&T) -> IK,
```

Comment:
This K -> IK change is quite sudden. I understand that `K` stands for `Key`, but is `IK` an `IndexKey`? 
I think it would be good to have some clear reference to it... I doubt it will be very clear to any people diving into this code from outside.

Reply:
It's a way to differentiate it from `PK`, and a generic `K` that could stand for any of them.